### PR TITLE
updating decimals

### PR DESCRIPTION
**Pull request by karthikeyan-cere**

### CODE REVIEW

File: src/modules/friendly-bot/friendly-bot.service.ts [hunk 1]
```diff
@@ -71,12 +71,14 @@ export class FriendlyBotService implements FriendlyBotServiceInterface {
...
     const networkParam = this.networkParams.find((item) => item.type === network);
     const {balance} = await this.getBalance(destination, network);
-    const initialBal = +balance / 10 ** 15;
+    const decimal = network === "TESTNET" ? 15 : 10;
+    const initialBal = await formatBalance(balance, {decimals: decimal});
     this.logger.debug(`Initial Balance: ${initialBal}`);
     const value = await this.configService.get('NUMBER_OF_TOKENS_TO_SEND');
-    const maxBalance = Number(this.configService.get('MAX_BALANCE'));
+    const actualValue = +value * 10 ** decimal;
+    const maxBalance = Number(this.configService.get('MAX_BALANCE')) * 10 ** decimal;
```
Comment (AndreiNavoichyk) [inline src/modules/friendly-bot/friendly-bot.service.ts:79]: Decimals for mac balance should use the correct decimals.

File: src/modules/friendly-bot/friendly-bot.service.ts [hunk 2]
```diff
@@ -71,12 +71,14 @@ export class FriendlyBotService implements FriendlyBotServiceInterface {
     }
     const networkParam = this.networkParams.find((item) => item.type === network);
     const {balance} = await this.getBalance(destination, network);
-    const initialBal = +balance / 10 ** 15;
+    const decimal = network === "TESTNET" ? 15 : 10;
```
Comment (AndreiNavoichyk) [inline src/modules/friendly-bot/friendly-bot.service.ts:74]: Create a JIRA ticket to remove this hardcode after we will update/get rid of TESTNET. Put JIRA id here as ToDo.

File: src/modules/block-scanner/block-scanner.service.ts [hunk 3]
```diff
@@ -240,9 +240,8 @@ export class BlockScannerService implements BlockScannerServiceInterface {
     const {
       data: {free: balance},
     } = await networkProp.api.query.system.account(address);
-    // FIXME: Fix the decimal position once it is fixed in chain
-    // const decimal = await this.api.registry.chainDecimals;
-    const result = await formatBalance(balance, {decimals: 15});
+    const decimal = network === "TESTNET" ? 15: 10;
```
Comment (AndreiNavoichyk) [inline src/modules/block-scanner/block-scanner.service.ts:243]: Create a JIRA ticket to remove this hardcode after we will update/get rid of TESTNET. Put JIRA id here as ToDo.

File: src/modules/friendly-bot/friendly-bot.service.ts (hunk 1 above)
Comment (karthikeyan-cere) [reply src/modules/friendly-bot/friendly-bot.service.ts:80]: Yes, I am using the correct decimals. Based on the decimals will be multiplied to the maxBalance.

**Review by karthikeyan-cere — COMMENTED** [review]
Updated the task number

File: src/modules/block-scanner/block-scanner.service.ts (hunk 3 above)
Comment (karthikeyan-cere) [reply src/modules/block-scanner/block-scanner.service.ts:244]: updated the task link

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**